pyphoon/main.py

```diff
@@ -0,0 +1,159 @@
+"""
+Copyright (c) 2017 Riptide IO, Inc. All Rights Reserved.
```

Comment:
What?

Also: Why breaking such a small script into two files at all, and, if you think its necessary, why calling the file with the __main__ "function" pyphoon and the file without it main.py?